src/components/dictionaryConcepts/components/helperFunction.js

```diff
@@ -62,3 +62,4 @@ export const MAP_TYPE = {
 export const TRADITIONAL_OCL_HOST = urlConfig.TRADITIONAL_OCL_HOST;
 
 export const CUSTOM_SOURCE = 'Custom';
+export const SELECT_EVENT_NAME = 'source';
```

Comment:
Is this a JavaScript event named source?

Reply:
No, it is not, I have changed the name of the constant.